src/helpers/disk_io.cc

```diff
@@ -0,0 +1,194 @@
+/*
```

Comment:
@lcyyork : are these still needed? I assume that the I/O in ambit should be enough now.

Reply:
> @lcyyork : are these still needed? I assume that the I/O in ambit should be enough now.

I moved the read/write a vector of double to this file. These two functions were used to read and write RDMs before. I guess we no longer need these two functions.